src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -277,14 +270,49 @@ bool OPT_AppelDuSimplexe(const OptimizationOptions& options,
             ProbSpx = nullptr;
             solver = nullptr;
```

Comment:
`ProbSpx` and `solver` are local pointers. It's quite useless to set them to `nullptr` at the end of this function.